webdriver-spec.html

```diff
@@ -3941,8 +3941,9 @@
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>If the <a>current top-level browsing context</a> is <a>no longer open</a>,
-  return <a>error</a> with <a>error code</a> <a>no such window</a>.
+ <li><p>If the <a>current browsing context</a> is <a>no longer
+  open</a>, return <a>error</a> with <a>error code</a> <a>no such
+  window</a>.
```

Comment:
Yes, in this case it makes sense to check the current browsing context as opposed to the _top-level_ browsing context: when the user has not switched to a frame, the current browsing context is equal to the top-level browsing context implicitly. (-: